eslint.config.js

```diff
@@ -0,0 +1,160 @@
+import comments from "@eslint-community/eslint-plugin-eslint-comments/configs";
+import eslint from "@eslint/js";
+import stylisticTs from "@stylistic/eslint-plugin-ts";
+import jsdoc from "eslint-plugin-jsdoc";
+import jsonc from "eslint-plugin-jsonc";
+import markdown from "eslint-plugin-markdown";
+import n from "eslint-plugin-n";
+import noOnlyTests from "eslint-plugin-no-only-tests";
+import perfectionist from "eslint-plugin-perfectionist";
+import regexp from "eslint-plugin-regexp";
+import vitest from "eslint-plugin-vitest";
+import yml from "eslint-plugin-yml";
+import globals from "globals";
+import tseslint from "typescript-eslint";
+
+export default tseslint.config(
+	// Global ignores
```

Comment:
[Style] "Nit" (nitpicking): the fact that it's an `ignores` on its own means it's a global ignores:

```suggestion
```